src/plugins/kibana_react/public/code_editor/languages/css/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Sorry if someone already asked this; what is the reason for having these language definitions live here and not in the `@kbn/monaco` package (with painless etc)?

Reply:
I'm not sure if it is a good idea to put some custom languages, required only in kibana plugins, to the core package. If the code owner will say, that it is a good idea, I'll do it)

Reply:
What do you think about it? Is there any reasons for doing it?

Reply:
Right, as far as I understand it, the languages defined in kbn/monaco are intended for use in plugins, or at least that plugins are the biggest consumers of that functionality. Could you help me understand the distinction with, for e.g., Handlebars vs Painless?

IMO it would be best to have a single place where languages are created and registered for easy reuse and discoverability - similar to @poffdeluxe comment.

Is part of the concern JS bundle size?

With the current kbn/monaco package plugins can't cherry-pick/declare languages they want to use and there is no good way to audit what languages are being used where (searching I suppose). But with monaco language definitions I don't think there is _a lot_ of JS being loaded typically.

Reply:
Painless, xjson and Esql are specific for elastic stack. But markdown, css and handlebars are common and they are just imported from monaco. I was thinking about that languages, which require syntax implementation, need to be located at kbn/monaco, all other - at plugins. But if somebody want to move languages, please, feel free to do it) I'm sorry, right now I'm working on the other task.